source/manual/howto-ssh-to-machines-in-aws.html.md

```diff
@@ -46,7 +46,7 @@ ip-10-1-6-88.eu-west-1.compute.internal
 
 SSH with agent forwarding to the jumpbox:
 
-        ssh -A friendlygiraffe@integration
+        ssh -A <EXAMPLEUSER>@integration
```

Comment:
perhaps `$USERNAME` instead of `<EXAMPLEUSER>` since that should just work even if copy-and-pasted.